readme.md

```diff
@@ -161,6 +161,7 @@
 - [DuckDuckGPT](https://github.com/kudoai/duckduckgpt) - Brings the magic of ChatGPT to search results.
 - [ChatGPT Infinity](https://github.com/adamlui/chatgpt-infinity) - Makes ChatGPT automatically answer random questions to increase your knowledge.
 - [ChatGPT Auto-Continue](https://github.com/adamlui/chatgpt-auto-continue) - Automatically clicks "Continue generating" when responses are cut off.
+- [GoogleGPT](https://www.googlegpt.io) - Adds ChatGPT to Google Search.
```

Comment:
Link to the GitHub repo.

Reply:
Ok it's updated (the prev URL was a redir to GH repo)